packages/library/src/components/bal-select-option/bal-select-option.scss

```diff
@@ -2,9 +2,12 @@
 @import 'node_modules/bulma/sass/components/dropdown.sass';
 
 :host {
-  display: block;
   font-family: $family-primary !important;
 
+  &.is-connected {
+    display: block;
```

Comment:
May using `initial`

Reply:
this class is added after the component is connected